### ISSUE TITLE
Builds are failing on master

### ISSUE THREAD

**Issue by billsacks** [ESMCI/cime#1656]
From the latest cime master, I tried the following on yellowstone:

```
./create_test ERP.f45_g37_rx1.A.yellowstone_intel
```

This died with a bunch of compilation errors; the first few were:

```
        /glade/p/work/sacks/cime/src/components/data_comps/dice/dice_comp_mod.F90(8): error #7002: Error in opening the compiled module file.  Check INCLUDE paths.   [SHR_CONST_MOD]

        /glade/p/work/sacks/cime/src/components/data_comps/dice/dice_comp_mod.F90(10): error #7002: Error in opening the compiled module file.  Check INCLUDE paths.   [SHR_SYS_MOD]

        /glade/p/work/sacks/cime/src/components/data_comps/dice/dice_comp_mod.F90(19): error #7002: Error in opening the compiled module file.  Check INCLUDE paths.   [MCT_MOD]
```

**Comment by billsacks** [comment]
This build works in 8517405

**Comment by jedwards4b** [comment]
FIxed with commit 9a43e4b, pushed directly to master.
